src/server/game/Handlers/MiscHandler.cpp

```diff
@@ -1191,7 +1191,7 @@ void WorldSession::HandleSelectFactionOpcode(WorldPackets::Misc::FactionSelect&
 
 void WorldSession::HandleSetWarModeOpcode(WorldPackets::Misc::SetWarMode& warMode)
 {
-    uint32 const warModeSpellId = 282559; // Enlisted
+    uint32 const warModeSpellId = 269083; // Enlisted
```

Comment:
Not sure about that one, did you check in sniffs ?

Reply:
no, but this spell is fully hung on the player and works well) I'm not sure what it is blizzlike

test in game good

Reply:
Checked in sniffs, it's still 282559